build/templates/electron_version.tmpl

```diff
@@ -31,4 +31,4 @@
 
 #define ELECTRON_VERSION "v" ELECTRON_VERSION_STRING
```

Comment:
now that this is autogenerated we can get rid of these silly preprocessor hacks and just write

```cpp
#define ELECTRON_VERSION "v$major.$minor.$patch"
```
or so?

or even just add the full semver to the keys and write

```cpp
#define ELECTRON_VERSION "v$semver"
```